README.md

```diff
@@ -153,11 +158,45 @@ router Docker image is set to the image built and push from the previous step.
 
 ```yaml
 # config-dev.yaml
+BatchRunnerConfig:
+  TimeInterval: 10s
 DbConfig:
   User: turing
   Password: turing
 DeployConfig:
   EnvironmentType: dev 
+EnsemblingJobConfig:
+  DefaultEnvironment: dev
+  BatchSize: 10
```

Comment:
Batch size refers to how many records get processed in the batch job. It is used in the List method because we are polling for a number of records to be processed at once.

https://github.com/ashwinath/turing/blob/batch-ensembler-orchestration/api/turing/batch/runner/ensembling.go#L188

Reply:
Also, your link from the comment above already points to something else :)

Reply:
I see where the confusion is coming from, I'm getting confused already but I think there are some terms we need to clarify. 

Ensembling Job -> 1 spark application running the Ensembler.
Batch Job -> multiple ensembling jobs batched into multiple iterations.

I didn't think of the ensembling job as a batch job by itself because all it does is to ensemble multiple datasets into one.

Nonetheless it does prove to be very confusing so I'm changing the name to `RecordsToProcessInOneIteration`.